lib/Analysis/GateAnalysis.cc

```diff
@@ -230,15 +223,32 @@ void GateAnalysisImpl::processPhi(PHINode *PN, Instruction *insertionPt) {
 
       // Or the successors unconditionally flows to an already processed block.
       // Note that there can be at most one such block.
-      for (auto &BlockValuePair : flowingValues) {
-        if (m_PDT.dominates(BlockValuePair.first, S)) {
-          SuccToVal[S] = BlockValuePair.second;
-          GSA_LOG(errs() << "2) SuccToVal[" << S->getName()
-                         << "] = postdom for cd "
-                         << BlockValuePair.first->getName() << ": "
-                         << SuccToVal[S]->getName() << "\n");
+      bool found = false;
+      BasicBlock *bb =  S;
```

Comment:
nit: Can you give more descriptive names to the variables here and format the code with clang-format?